src/Components/Favorites/Favorites.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./Favorites.css";
+import FavoriteCard from "../FavoriteCard/FavoriteCard";
+
+const Favorites = ({ favorites, setFavorites }) => {
```

Comment:
Until now i did not know we could pass useState as props. Good job passing it destructured and assigning the correct prop type to it. I will update my logic with this since it is more efficient/concise